src/Component/FieldTypes/FieldTypeInterface.php

```diff
@@ -23,10 +23,8 @@ interface FieldTypeInterface
      * $options.
      *
      * @param mixed $data
-     *
-     * @return string
      */
-    public function render(Field $field, $data, array $options);
+    public function render(Field $field, $data, array $options): string;
```

Comment:
I believe indeed it is a BC-break :( I would skip that for now, unfortunately. There is a part of me that says "let's just document it and move on with the modern syntax" but I'm not sure... cc @lchrusciel I need a second opinion 🖖

Reply:
I must agree :/ Perhaps we can sooner than later just release GrudBundle 2.0

Reply:
Removed. But I believe we can leave the extending classes with that return type, can we ?